client/signup/steps/site-information/index.jsx

```diff
@@ -0,0 +1,181 @@
+								<FormLabel htmlFor="name">
+									{ siteNameLabelText }
+									<InfoPopover className="site-information__info-popover" position="top">
+										{ translate( 'This will be used for the title of your site.' ) }
+									</InfoPopover>
+								</FormLabel>
+								<FormTextInput
+									id="name"
+									name="name"
+									placeholder={ translate( 'eg. Google, Automattic, AirBnb' ) }
+									onChange={ this.handleInputChange }
+									value={ this.state.name }
+								/>
+							</FormFieldset>
+
+							{ isBusinessSiteSelected && (
+								<Fragment>
+									<FormFieldset>
+										<FormLabel htmlFor="address">
+											{ translate( 'Address' ) }
+											<InfoPopover className="site-information__info-popover" position="top">
+												{ translate( 'Where can people find your business?' ) }
+											</InfoPopover>
+										</FormLabel>
+										<FormTextarea
+											id="address"
+											name="address"
+											placeholder={ 'eg. 21 Main street\nOttawa  ON\nK1V 2K5' }
+											onChange={ this.handleInputChange }
+											value={ this.state.address }
+										/>
+									</FormFieldset>
+									<FormFieldset>
+										<FormLabel htmlFor="phone">
+											{ translate( 'Phone number' ) }
+											<InfoPopover className="site-information__info-popover" position="top">
+												{ translate( 'How can people contact you?' ) }
+											</InfoPopover>
+										</FormLabel>
+										<FormTextInput
+											id="phone"
+											name="phone"
+											placeholder={ translate( 'eg. (613) 425-0183' ) }
+											onChange={ this.handleInputChange }
+											value={ this.state.phone }
+										/>
+									</FormFieldset>
+								</Fragment>
+							) }
+							<div className="site-information__submit-wrapper">
+								<Button primary={ true } type="submit">
+									{ translate( 'Continue' ) }
+								</Button>
+							</div>
+						</Card>
+					</form>
+				</div>
+			</div>
+		);
+	}
+
+	render() {
+		const { flowName, positionInFlow, signupProgress, stepName, translate } = this.props;
+
+		return (
+			<StepWrapper
+				flowName={ flowName }
+				stepName={ stepName }
+				positionInFlow={ positionInFlow }
+				headerText={ translate( 'Almost done, just a few more things.' ) }
+				subHeaderText={ translate( "We'll add this information to your new website." ) }
+				signupProgress={ signupProgress }
+				stepContent={ this.renderContent() }
+				backUrl={ '/start/main-onboarding/business-needs' }
+			/>
+		);
+	}
+}
+
+/**
+ * Returns translated text for the name field depending on the site type
+ *
+ * @param   {String} siteType E.g,'business', 'blog'
+ * @returns {String|Object} TranslatableString
+ */
+function getSiteNameLabelText( siteType ) {
+	switch ( siteType ) {
+		case 'business':
+			return i18n.translate( 'Business name' );
+		case 'blog':
+			return i18n.translate( 'Blog name' );
+	}
+	return i18n.translate( 'Site name' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'SITE NAME' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).